accounts/abi/bind/backends/simulated_test.go

```diff
@@ -274,7 +285,10 @@ func TestSimulatedBackend_NonceAt(t *testing.T) {
 	}
 
 	// create a signed transaction to send
-	tx := types.NewTransaction(nonce, testAddr, big.NewInt(1000), params.TxGas, big.NewInt(1), nil, nil, nil, nil)
+	// head, _ := sim.HeaderByNumber(context.Background(), nil) // Should be child's, good enough
```

Comment:
i've seen this line commented several times. Is that us? Is that on purpose?

Reply:
I commented it out. It was previously used to get the basefee